### PR TITLE
bitrise-step-xctest-cobertura-xml 1.0.0

### PR DESCRIPTION
**Pull request by ollitapa**
TagCheck

Hi, here is a new small step to generate Cobertura formatted XML-file from xcresult bundle produced by Xcode 11. The file can be then uploaded for example to SonarQube. 

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- cordova
+- ionic
+type_tags:
+- utility
+- test
+toolkit:
+  go:
+    package_name: github.com/ollitapa/bitrise-step-xctest-cobertura-xml
+is_requires_admin_user: false
+is_always_run: false
```
Comment (lpusok) [inline steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Probably need to set Is_always_run to true to export xml even if for example the test step fails.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+title: XCTest result to Cobertura XML
```
Comment (lpusok) [inline steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Would recommend the title `Convert XCTest result to Cobertura XML`, to include the action done.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  commit: a7c4241cd36da299b4093948bf6097f9c0e65fa2
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+- xamarin
+- react-native
+- cordova
+- ionic
```
Comment (lpusok) [inline steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Could add `flutter` project type.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    is_expand: true
+    is_required: true
+    summary: Path to XCTest result bundle
+    title: Path to XCTest result bundle
+    value_options: []
+  path_to_xcresult: $BITRISE_XCRESULT_PATH
+- opts:
+    description: |
+      Directory where to put the resulting `coverage.json` and `cobertura.xml`
+      Defualt is `$BITRISE_DEPLOY_DIR`
```
Comment (lpusok) [inline steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Typo in 'Defualt'.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- utility
```
Comment (lpusok) [inline steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: The utility type will suffice, test is used only for the steps that are test runners.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml (hunk 1 above)
Comment (ollitapa) [reply steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Ah, yes, fixed.

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml (hunk 2 above)
Comment (ollitapa) [reply steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Sure, thanks!

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml (hunk 4 above)
Comment (ollitapa) [reply steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Fixed

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml (hunk 5 above)
Comment (ollitapa) [reply steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Fixed

File: steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml (hunk 3 above)
Comment (ollitapa) [reply steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml]: Oh, ok, didn't think of that since flutter wasn't in the template.

**Pull request merged**